exporter/datadogexporter/metrics_exporter.go

```diff
@@ -141,6 +142,7 @@ func (exp *metricsExporter) pushSketches(ctx context.Context, sl sketches.Sketch
 	if isMetricExportV2Enabled() {
 		resp, err = exp.metricsAPI.Client.Cfg.HTTPClient.Do(req)
 	} else {
+		exp.params.Logger.Warn("You are using the deprecated Zorkian codepath that will be removed in the next release; use the metrics serializer instead: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/README.md")
```

Comment:
This will generate a warning every time a distribution metric is pushed. IMO it's too verbose so propose to either sample it or remove here